comm/src/test/scala/coop/rchain/comm/transport/TransportLayerRuntime.scala

```diff
@@ -66,13 +66,14 @@ abstract class TransportLayerRuntime[F[_]: Monad, E <: Environment] {
             remoteTl <- createTransportLayer(e2)
             local    = e1.peer
             remote   = e2.peer
+            _        <- localTl.receive(null, null)
             _ <- remoteTl.receive(
                   protocolDispatcher.dispatch(remote),
                   streamDispatcher.dispatch(remote)
                 )
             r <- execute(localTl, local, remote)
             // arbitrary sleep value, so environment has time to handle requests
-            _ <- time.sleep(200 millisecond)
+            _ <- time.sleep(1000 millisecond)
```

Comment:
This sleep looks bad. I know it was here before but it still looks bad. Is this really needed? Why 1s?

Reply:
@odeac This is a unit test environment. I'll take a look at this in a different task because of nondeterminism.